spikeinterface/toolkit/postprocessing/template_tools.py

```diff
@@ -150,16 +223,27 @@ def get_template_extremum_channel_peak_shift(waveform_extractor, peak_sign='neg'
         elif peak_sign == 'pos':
             peak_pos = np.argmax(template[:, chan_ind])
         shift = peak_pos - waveform_extractor.nbefore
-        shifts[unit_id] = shift
+        shifts[unit_id] = shift.astype("int64")
```

Comment:
Why forcing to int64 ? It will cuase problem for parralisation. It is a small int (some sample)